src/processors/default/processTrackArtworks.ts

```diff
@@ -0,0 +1,115 @@
+import { readFile, writeFile, rm } from 'node:fs/promises';
+import http from 'node:http';
+import https from 'node:https';
+import Path from 'node:path';
+
+import im from 'imagemagick';
+import { create } from 'ipfs-http-client';
+import tempDirectory from 'temp-dir';
+
+import { Table } from '../../db/db';
+import { missingProcessedArtworks } from '../../triggers/missing';
+import { Clients } from '../../types/processor';
+import { ProcessedTrack } from '../../types/track';
+import { rollPromises } from '../../utils/rollingPromises';
+
+const name = 'addMetadataIPFSHash';
+
+function imageBuffer(url: string): Promise<Buffer> {
```

Comment:
i used raw http clients here, just because wanted it in a buffer, not familiar with axios